dashboard/app/src/java/org/sakaiproject/dash/app/SakaiProxy.java

```diff
@@ -82,8 +82,23 @@
 	public static final String EVENT_MODIFY_CALENDAR_EVENT_EXCLUDED = CalendarService.EVENT_MODIFY_CALENDAR_EVENT_EXCLUDED;
 	public static final String EVENT_MODIFY_CALENDAR_EVENT_EXCLUSIONS = CalendarService.EVENT_MODIFY_CALENDAR_EVENT_EXCLUSIONS;
 	
+    /* Here are the samigo events we care about
+       sam.assessment.publish | siteId=f08b61c2-2b74-4355-8978-ae26718a00b7, assessmentId=9, publishedAssessmentId=13 
+       sam.pubsetting.edit    | siteId=f08b61c2-2b74-4355-8978-ae26718a00b7, publishedAssessmentId=13 
+       sam.pubassessment.confirm_edit | siteId=f08b61c2-2b74-4355-8978-ae26718a00b7, publishedAssessmentId=13 
+       sam.pubassessment.republish    | siteId=f08b61c2-2b74-4355-8978-ae26718a00b7, publishedAssessmentId=13  
+       sam.pubAssessment.remove       | siteId=f08b61c2-2b74-4355-8978-ae26718a00b7, publisedAssessmentId=13  
+    */
+
+	public static final String EVENT_PUBASSESSMENT_PUBLISH = "sam.assessment.publish";
```

Comment:
Yeah, I agree with this that they should be part of the samigo api, we should add these on a separate issue and clean them up.